JoomGalleryfriends/JG4-dev
# Migration service V2

This PR adds migration for Users, Favourites, Votes and Comments.

The PR applies changes to the DB such that the following DB structure is followed and old records are migrated into this DB structure:
https://docs.google.com/presentation/d/1kXGfGRrHswU0M3yh0zUvOwW1fYqB07cksHzNxzcEyxI/edit?pli=1#slide=id.g1b778661e29_0_0

### How to test this PR

Start a new migration and try to migrate all possible content types.

Images and Categories should work the same way as before. Users, Favourites, Votes and Comments are not visible in the GUI after migration and must be checked on the database using e.g phpMyAdmin.

![MigrationManager](https://github.com/user-attachments/assets/064a1d99-14ed-4466-a7cd-5a5e28504673)
